save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/Plugin.kt

```diff
@@ -202,14 +207,14 @@ abstract class Plugin(
 
     /**
      * Returns a sequence of directories, where resources for this plugin may be located.
-     * This takes into account, that if underlying directory contains it's own SAVE config,
+     * This takes into account, that if underlying directory contains its own SAVE config,
      * then this plugin shouldn't touch these resources; it should be done by plugins from that config.
      *
      * @return a sequence of directories possibly containing this plugin's test resources
      */
     private fun Path.resourceDirectories(): Sequence<Path> = findDescendantDirectoriesBy(true) { file ->
         // this matches directories which contain their own SAVE config
-        fs.metadata(file).isRegularFile || fs.list(file).none { it.isSaveTomlConfig() }
+        fs.metadata(file).isDirectory && fs.list(file).none { it.isSaveTomlConfig() }
```

Comment:
how it worked before? that's a different logic now

Reply:
It was working, because inside `findDescendantDirectoriesBy` we filter only directories, as the name of the method assumes. So the predicate was passing, but on the very next step the file was filtered out. So I changed the predicate to make it more consistent.